src/components/filterModal/statsList/BaseEditorLogarithmic.vue

```diff
@@ -51,29 +54,47 @@ export default {
         return {
             min: MIN,
             max: MAX,
-            selectedMin: this.preselectedMin || MIN,
-            selectedMax: this.preselectedMax || MAX,
+            currentMin: this.preselectedMin,
+            currentMax: this.preselectedMax,
+            selectedMin: this.closestLeftMark(this.preselectedMin),
+            selectedMax: this.closestRightMark(this.preselectedMax),
+            key: 0,
         };
     },
     methods: {
         addData() {
             this.onSubmit(this.selectedMin, this.selectedMax);
         },
         changeValues([min, max]) {
-            this.selectedMin = min;
-            this.selectedMax = max;
+            this.currentMin = min;
+            this.currentMax = max;
+            this.selectedMin = (min < this.preselectedMin ? this.closestLeftMark(this.preselectedMin) : this.closestLeftMark(min));
```

Comment:
You send this.selectedMin, this.selectedMax values on Submit, but it's only a scale representation. What's the sense of inputs?